README

```diff
@@ -18,7 +18,7 @@ Backup Format
 DSpace content is replicated / backed up as a set of AIPs (Archival Information Packages).
 The Replication Task Suite supports two different AIP formats:
   1. The DSpace AIP Format based on METS (Default)
-     https://wiki.duraspace.org/display/DSDOC3x/DSpace+AIP+Format
+     https://wiki.lyrasis.org/display/DSDOC3x/DSpace+AIP+Format
```

Comment:
Good idea! As long as we're updating links, maybe also point to a current version: https://wiki.lyrasis.org/display/DSDOC6x/DSpace+AIP+Format

Reply:
Yeah, wow. Didn't notice it was that old.